src/test/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/AnVIL_WGS_100_exported_cohort/sample_list

```diff
@@ -0,0 +1,100 @@
+30,HG00561
```

Comment:
probably could use just a few samples, and use a subsetted interval list, so that the test files can be small. there's a small interval list in the CreateVariantIngestFiles integration test resources, if that helps!

Reply:
Yes please!  Just take a few ( < 5 ) samples and just a few sites (100s)